packages/react-urql/examples/1-getting-started/README.md

```diff
@@ -0,0 +1,34 @@
+## React & service worker example
+
+This shows how to use URQL in the browser with React and a service worker.
+
+A page displays a to-do list, and you can add items or mark them as complete. The to-dos are saved in a local IndexedDB database so will persist across sessions.
+
+## Usage
+
+#### 1. Install dependencies in repo root
+
+To get started with the example, make sure to install depdencies for `urql` and `react-urql` directories. Running `yarn build` in the root will also go into each `packages/*` folder, install dependencies, and build. (The example's `package.json` will create a symlink to the built `react-urql`.)
+
+```bash
+# In root directory
+yarn
+yarn build
```

Comment:
This should have been done from the get go - thanks so much for sorting this out!

How would you feel about doing this in a postInstall script for each example? That way users could just do `yarn` and `yarn start` from the working directory.

Reply:
Done.